### PR TITLE
"Sort" keyword missing in verb-noun pattern

### PR DESCRIPTION
**Pull request by kborowinski**
"Sort" keyword missing in verb-noun pattern causing "Sort-Object" to not being highlighted correctly.

### CODE REVIEW

**Comment by omniomi** [comment]
I will add this to the list of items on my fix list. Thanks!

File: PowerShellSyntax.tmLanguage [hunk 1]
```diff
@@ -414,7 +414,7 @@
...
+					<string>(?:(\p{L}|\d|_|-|\\|\:)*\\)?\b(?i:Add|Approve|Assert|Backup|Block|Checkpoint|Clear|Close|Compare|Complete|Compress|Confirm|Connect|Convert|ConvertFrom|ConvertTo|Copy|Debug|Deny|Disable|Disconnect|Dismount|Edit|Enable|Enter|Exit|Expand|Export|Find|Format|Get|Grant|Group|Hide|Import|Initialize|Install|Invoke|Join|Limit|Lock|Measure|Merge|Mount|Move|New|Open|Optimize|Out|Ping|Pop|Protect|Publish|Push|Read|Receive|Redo|Register|Remove|Rename|Repair|Request|Reset|Resize|Resolve|Restart|Restore|Resume|Revoke|Save|Search|Select|Send|Set|Show|Skip|Sort|Split|Start|Step|Stop|Submit|Suspend|Switch|Sync|Test|Trace|Unblock|Undo|Uninstall|Unlock|Unprotect|Unpublish|Unregister|Update|Use|Wait|Watch|Write)\-.+?(?:\.(?i:exe|cmd|bat|ps1))?\b</string>
```
Comment (tylerl0706) [inline PowerShellSyntax.tmLanguage]: I actually don't see Build or Deploy on this list (these were added in PS Core 6), would you mind adding them?

Comment (tylerl0706) [reply PowerShellSyntax.tmLanguage]: I think those are the only 2... might be worth doing a `Get-Verb` to double check.

Comment (omniomi) [reply PowerShellSyntax.tmLanguage]: Quick 'n dirty: `z:\> (Get-Verb).Verb -join '|'`

Comment (tylerl0706) [reply PowerShellSyntax.tmLanguage]: Sort isn't an approved verb 😱 

```
Add|Approve|Assert|Backup|Block|Build|Checkpoint|Clear|Close|Compare|Complete|Compress|Confirm|Connect|Convert|ConvertFrom|ConvertTo|Copy|Debug|Deny|Deploy|Disable|Disconnect|Dismount|Edit|Enable|Enter|Exit|Expand|Export|Find|Format|Get|Grant|Group|Hide|Import|Initialize|Install|Invoke|Join|Limit|Lock|Measure|Merge|Mount|Move|New|Open|Optimize|Out|Ping|Pop|Protect|Publish|Push|Read|Receive|Redo|Register|Remove|Rename|Repair|Request|Reset|Resize|Resolve|Restart|Restore|Resume|Revoke|Save|Search|Select|Send|Set|Show|Skip|Split|Start|Step|Stop|Submit|Suspend|Switch|Sync|Test|Trace|Unblock|Undo|Uninstall|Unlock|Unprotect|Unpublish|Unregister|Update|Use|Wait|Watch|Write
```

That explains why it's not in the list in the first place.

p.s. needed to alpha-sort the results of that line @omniomi just to share knowledge

Comment (kborowinski) [reply PowerShellSyntax.tmLanguage]: Yes, oddly *Sort* is missing from approved verb in PS s, but I think we should add it anyways? I'll also add *Deploy* and *Build* verbs.

Comment (kborowinski) [reply PowerShellSyntax.tmLanguage]: May I update the commit with this list?

**Comment by omniomi** [comment]
I have tested @kborowinski's version and everything highlights without clobbering `foreach () {}`. Looks good to me.

cmdlets

Once we're all done squashing bugs we'll need to do an overall review of this section. There's some duplication elsewhere in the spec for some of the listed functions and questions about scoping but that's for the cleanup phase ;D

**Pull request merged**